plugins/modules/net_tools/nmcli.py

```diff
@@ -345,11 +455,140 @@
+                             Wi-Fi power saving), NM_SETTING_WIRELESS_POWERSAVE_IGNORE (1) (don't touch currently configure setting) or
+                             NM_SETTING_WIRELESS_POWERSAVE_DEFAULT (0) (use the globally configured value). All other values are reserved.
+                type: int
+                default: 0
+                choices: [ 0, 1, 2, 3 ]
+            rate:
+                description: If non-zero, directs the device to only use the specified bitrate for communication with the access point. Units are in Kb/s, ie
+                             5500 = 5.5 Mbit/s. This property is highly driver dependent and not all devices support setting a static bitrate.
+                type: int
+                default: 0
+            seen-bssids:
+                description: A list of BSSIDs (each BSSID formatted as a MAC address like "00:11:22:33:44:55") that have been detected as part of the Wi-Fi
+                             network. NetworkManager internally tracks previously seen BSSIDs. The property is only meant for reading and reflects the BSSID
+                             list of NetworkManager. The changes you make to this property will not be preserved.
+                type: list
+                elements: str
+            tx-power:
+                description: If non-zero, directs the device to use the specified transmit power. Units are dBm. This property is highly driver dependent and
+                             not all devices support setting a static transmit power.
+                type: int
+                default: 0
+            wake-on-wlan:
+                description: The NMSettingWirelessWakeOnWLan options to enable. Not all devices support all options. May be any combination of
+                             NM_SETTING_WIRELESS_WAKE_ON_WLAN_ANY (0x2), NM_SETTING_WIRELESS_WAKE_ON_WLAN_DISCONNECT (0x4),
+                             NM_SETTING_WIRELESS_WAKE_ON_WLAN_MAGIC (0x8), NM_SETTING_WIRELESS_WAKE_ON_WLAN_GTK_REKEY_FAILURE (0x10),
+                             NM_SETTING_WIRELESS_WAKE_ON_WLAN_EAP_IDENTITY_REQUEST (0x20), NM_SETTING_WIRELESS_WAKE_ON_WLAN_4WAY_HANDSHAKE (0x40),
+                             NM_SETTING_WIRELESS_WAKE_ON_WLAN_RFKILL_RELEASE (0x80), NM_SETTING_WIRELESS_WAKE_ON_WLAN_TCP (0x100) or the special values
+                             NM_SETTING_WIRELESS_WAKE_ON_WLAN_DEFAULT (0x1) (to use global settings) and NM_SETTING_WIRELESS_WAKE_ON_WLAN_IGNORE (0x8000) (to
+                             disable management of Wake-on-LAN in NetworkManager).
+                type: int
+                default: 1
+       version_added: 3.5.0
+    ignore_unsupported_suboptions:
+       description:
+            - Ignore suboptions which are invalid or unsupported by the version of NetworkManager/nmcli
+              installed on the host.
+            - Only 'wifi' and 'wifi_sec' options are currently affected.
+       type: bool
+       default: no
        version_added: 3.5.0
```

Comment:
```suggestion
       version_added: 3.6.0
```
3.5.0 has already been released.